src/webgpu/web_platform/reftests/ref/canvas_size_different_with_back_buffer_size-ref.html

```diff
@@ -0,0 +1,45 @@
+<html>
+  <title>WebGPU canvas_back_buffer_different_size (ref)</title>
+  <meta charset="utf-8" />
+  <link rel="help" href="https://gpuweb.github.io/gpuweb/" />
+  <canvas id="cvs_larger_than_back_buffer_ref" width="6" height="8"></canvas>
+  <canvas id="cvs_same_as_back_buffer_ref" width="3" height="4"></canvas>
+  <canvas id="cvs_smaller_than_back_buffer_ref" width="3" height="4"></canvas>
+  <script>
+  {
+    const context = cvs_larger_than_back_buffer_ref.getContext('2d');
+    context.fillStyle = "#FF0000";
+    context.fillRect(0, 0, 4, 4);
+    context.fillStyle = "#00FF00";
+    context.fillRect(4, 0, 2, 4);
+    context.fillStyle = "#0000FF";
+    context.fillRect(0, 4, 4, 4);
+    context.fillStyle = "#FFFF00";
+    context.fillRect(4, 4, 2, 4);
+    }
```

Comment:
To strictly match how we expect WebGPU to work, this canvas's inherent size should actually be 3x4, but its _CSS_ size should be 6px by 8px.